recipes-support/updater/files/sama5d27-som1-ek-sd/platform

```diff
@@ -1,5 +1,8 @@
 TITLE="SAM SOM"
 PLAT="sama5d27-som1-ek-sd"
+UPDATE_FILE_NAME="sama5d27"
+
+USB_DETECTION_DELAY=10
```

Comment:
10s is a lot maybe we can make it 3 or 5

Reply:
yeah, I use 4s on most projects. You originally had 10s in init, so not sure if that is needed on your end for testing.

Reply:
I had a usb stick from 2000 :)